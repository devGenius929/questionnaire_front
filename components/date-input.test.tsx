import React from 'react';
import renderer from 'react-test-renderer';
import DateInput from './date-input';

describe('Menu', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <DateInput
          id="dateTestId"
          name="dateTest"
          value="01021980"
          className="date"
          onChange={() => console.log('Date changed')}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
