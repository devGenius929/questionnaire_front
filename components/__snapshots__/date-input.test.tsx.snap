// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<input
  className="date"
  id="dateTestId"
  name="dateTest"
  onChange={[Function]}
  type="text"
  value="01021980"
/>
`;
